bossdata/remote.py

```diff
@@ -202,9 +202,18 @@ def get(self, remote_path, progress_min_size=10, auto_download=True):
         Raises:
             RuntimeError: File is not already mirrored and auto_download is False.
         """
-        local_path = self.local_path(remote_path)
-        if os.path.isfile(local_path):
-            return local_path
+
+        # Don't test if its a string; maybe later it won't be.  Test if it's a list
+        remote_paths = []
+        if not isinstance(remote_path, list):
```

Comment:
A tuple should also work here.  More generally, any iterable container is fine.  I believe you can identify any iterable using `hasattr(remote_path,__iter__)`.